nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -0,0 +1,185 @@
+{ config, lib, pkgs, buildEnv, ... }:
+
+with lib;
+
+let
+  cfg = config.services.healthchecks;
+  pkg = pkgs.healthchecks;
+  boolToPython = b: if b then "True" else "False";
+  environment = {
+    PYTHONPATH = pkg.pythonPath;
+    STATIC_ROOT = cfg.dataDir + "/static";
+    SECRET_KEY_FILE = cfg.secretKeyFile;
+    DB_NAME = "${cfg.dataDir}/healthchecks.sqlite";
+    ALLOWED_HOSTS = lib.concatStringsSep "," cfg.allowedHosts;
+    DEBUG = boolToPython cfg.debug;
+    REGISTRATION_OPEN = boolToPython cfg.registrationOpen;
+  } // cfg.settings;
```

Comment:
This makes me think you want a set of [freeform](https://nixos.org/manual/nixos/stable/index.html#sec-freeform-modules) module options instead.

Reply:
I did not know about them, thanks!